### DISCUSSION TITLE
Welcome to pcpartpicker Discussions!

### DISCUSSION THREAD

**Discussion by JonathanVusich** [JonathanVusich/pcpartpicker#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ghost** [comment]
Hi, is this still actively supported? I seem to be getting a dataset that is out of date, it doesn't have the latest components that I can see on on the website.

**Reply by JonathanVusich** [reply]
Unfortunately, PCPartPicker now has bot checks that prevent the scraper code from working properly. I am not experienced enough in building web scrapers to know how to bypass these checks, so the dataset is now going stale.
